artnet/plugin.yaml

```diff
@@ -5,21 +5,46 @@ plugin:
     description:
         de: 'Ansteuerung der meisten USB/Netwerk DMX Adapter'
         en: 'Can control most of USB/network DMX adapters'
-    maintainer: mode2k
+    maintainer: mode2k, jentz1986
     tester: ohinckel
-    state: qa-passed
+    state: ready
     keywords: dmx
 #    documentation: https://github.com/smarthomeNG/plugins/blob/develop/mqtt/README.md        # url of documentation (wiki) page
 
-    version: 1.3.0                 # Plugin version
-    sh_minversion: 1.3             # minimum shNG version to use this plugin
-#    sh_maxversion:                 # maximum shNG version to use this plugin (leave empty if latest)
-    multi_instance: False
-    restartable: unknown
```

Comment:
I think the `restartable` setting should be still listed there.

Reply:
OK, what are criteria for restartable plugins? Data is gone once plugin is restarted, but rest is stateless...

Reply:
check if your plugin will restart save after stop and then run method is called. then it can be set to True

Reply:
Guess I will implement and test the restarting feature. Was set to true in the YAML.

Reply:
This flag doesn't tell someone if a plugin restarts in the same state, it just tells you if the plugin can be restarted (some plugins has dependencies or even no code implemented to be restartable, in this case the flag is `False`).

But for this plugin it should be easy implement stop and start methods (just close the connection and reopen it again). If you want to restore the state you could also initialize the value based on the item's current values.